stanford/roles/cluster/defaults/main.yml

```diff
@@ -172,6 +177,31 @@ CLUSTER_SECURITY_GROUP_RULES:
       from_port: 22
       to_port: 22
       cidr_ip: "{{ DEPLOYMENT_CIDR }}"
+  emr:
+    - proto: icmp
+      from_port: 0
+      to_port: 65535
+      cidr_ip: "{{ DEPLOYMENT_CIDR }}"
+    - proto: tcp
+      from_port: 0
+      to_port: 65535
+      cidr_ip: "{{ DEPLOYMENT_CIDR }}"
+    - proto:  "tcp"
+      from_port:  "8443"
+      to_port:  "8443"
+      cidr_ip:  "204.246.160.32/28"
+    - proto:  "tcp"
+      from_port:  "8443"
+      to_port:  "8443"
+      cidr_ip:  "204.246.162.32/28"
+    - proto:  "tcp"
+      from_port:  "8443"
+      to_port:  "8443"
+      cidr_ip:  "54.240.198.0/24"
```

Comment:
Where exactly are these IPs coming from again?

This documentation [1] says they appear to be

> Various Amazon IP address ranges

used to

> Allows the cluster manager to communicate with the master nodes in each Amazon EMR cluster in a single VPC. 

Can you link to somewhere that lists these IP ranges?

[1] https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-man-sec-groups.html